### ISSUE TITLE
no such function: instr (code 1): on Android 4.4.2

### ISSUE THREAD

**Issue by mkeresztes** [mkeresztes/AndiCar#31]
```
Fatal Exception: java.lang.RuntimeException: Unable to resume activity {org.andicar2.activity/org.andicar2.activity.MainActivity}: android.database.sqlite.SQLiteException: no such function: instr (code 1): , while compiling: update gps_trackdetail set File = substr(File, instr(File, gps_track_id))
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3042)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3138)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2455)
       at android.app.ActivityThread.access$800(ActivityThread.java:164)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1362)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:5532)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:841)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:657)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by mkeresztes** [comment]
hotfix released in release 18.03-02

**Comment by mkeresztes** [comment]
published in 13.03-3
